### DISCUSSION TITLE
Your 570esp loader

### DISCUSSION THREAD

**Discussion by Huykiri0155P** [user202729/CasioEmu#8] [Q&A]
How to use your loader in UCF. I execute it by [AC][<][=] and that doesnt work:v

**Comment by Huykiri0155P** [comment]
![20231224_101417889](https://github.com/user202729/CasioEmu/assets/101462827/02daa986-c962-42d5-8eaf-f97a75630105)

**Comment by user202729** [comment]
Not sure, perhaps try to figure out how to run it on the emulator, then you can inspect the memory etc.

What is the loader supposed to do? The thread is long...

**Comment by Huykiri0155P** [comment]
Here is your loader:
https://community.casiocalc.org/topic/7583-fx-82-83gt-115-991es-plus-hacking/page-9#entry61746

**Reply by user202729** [reply]
I think the point of the loader is that if you proceed to continue typing more keys in, the values of the keys will be written somewhere in the memory in order to "load" more programs. I don't think it shows anything significant on the screen such as "Please press some keys" -- within 100 bytes there's barely any space to put things like that.
